### ISSUE TITLE
Typescript Definition File Not An External Module

### ISSUE THREAD

**Issue by pixelshaded** [inversify/InversifyJS#18]
```
declare module inversify {
```

should be  

```
declare module 'inversify' {
```

So that the following will compile

```
import inversify = require('inversify');
```

Right now its an ambient internal module and needs to be an ambient external module.

See 'Working with Other JavaScript Libraries' at http://www.typescriptlang.org/Handbook#modules

**Comment by remojansen** [comment]
Hi @pixelshaded this issue should be fixed now by commits f3e7859b9301f9ef7f73c5abdf79c6b451c08eaf and pull request https://github.com/borisyankov/DefinitelyTyped/pull/5698 

It would be much appreciated if you can conform that everything works as you expected. 

Thanks!

**Comment by remojansen** [comment]
I'm going to close this issue to ensure that we only have one unique issue to refer to this problem. You can follow the updates at #46
